Dependencies/xr/Source/ARKit/XR.mm

```diff
@@ -135,6 +160,18 @@ - (void) SetPlaneDetectionEnabled:(bool)enabled {
     planeDetectionEnabled = enabled;
 }
 
+- (bool) TrySetMeshDetectorEnabled:(const bool)enabled {
+    meshDetectionEnabled = enabled;
+    return enabled;
```

Comment:
>enabled [](http://example.com/codeflow?start=11&length=7)

these TrySetMeshDetectorEnabled should always return true not necessarily enabled/disabled value.